portal-backend/depmap/compound/api.py

```diff
@@ -20,3 +21,28 @@ def get(self):
         )
 
         return dose_curve_info
+
+
+@namespace.route("/prioritized_dataset")
+class PrioritizedDataset(Resource):
+    def get(self):
+        # TODO: Probably need to use this eventually to make sure the dataset has the compound
+        # compound_id = request.args.get("compound_id")
+
+        # TODO: How to figure out which dataset to prioritize? Right now only OncRef is supported,
+        # so don't worry about this yet.
```

Comment:
This is a good question, and it's one we're still trying to iron out to some extent (more discussion [here](https://app.asana.com/1/9513920295503/task/1210800769249818/comment/1210826445706204?focus=true) if you're interested). 

Even though it's not important for this PR, I'll just mention that since I believe you're only using breadbox datasets, I think it would be fine to order them by their "priority" field (which would essentially make Oncref the default).